babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
```

Comment:
Please remove comments from your codebase. This is to facilitate a clean build/programming.